packages/ui/src/organisms/Hero/Hero.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { forwardRef } from 'react'
+import type { HTMLAttributes } from 'react'
+
+export interface HeroProps extends HTMLAttributes<HTMLDivElement> {
+  /**
+   * ID to find this component in testing tools (e.g.: cypress,
+   * testing-library, and jest).
+   */
+  testId?: string
+  children: React.ReactNode
```

Comment:
It seems we can skip this! Both `HeroHeading` and `HeroImage` don't have/need it as well.
```suggestion
```